HorizonEditor/src/ContentBrowser/ContentBrowser.cpp

```diff
@@ -141,7 +138,14 @@ void ContentBrowser::OnImGuiRender()
 			}
 			else {
 				m_CurrentTexturePath = "";
-				ImGui::ImageButton((ImTextureID)icon->getId(), { thumbnailSize, thumbnailSize }, { 0, 1 }, { 1, 0 });
+				ImGui::ImageButton(filenameString.c_str(), (ImTextureID)icon->getId(), { thumbnailSize, thumbnailSize }, { 0, 1 }, { 1, 0 });
+
+				if (ImGui::IsItemHovered() && ImGui::IsMouseClicked(ImGuiMouseButton_Right))
+				{
+					HZN_WARN("item");
+					itemWasSelected = true;
+					selected_file = filenameString;
```

Comment:
selected_file should be the path not the name of file

Reply:
yeah I would fix those changes.